Gitbook deployment guide (#422)
klaytn/klaytn-docs: docs/smart-contract/deploy-guide.md

    }
}
```

> - Select Compiler in the icon panel. Choose the desired EVM environment. For the Klaytn networks, you can choose between Baobab (testnet) and Cypress (mainnet). And click Compile.

Reviewer: ```suggestion
- Select Compiler in the icon panel. Choose the desired EVM environment. For the Klaytn networks, you can choose between Baobab (testnet) and Cypress (mainnet). Click `Compile` when the sample code is ready to be complied before actual deployment.
```

![](img/deploy-with-ide/02_deployment_compile.png)

> - Now we can deploy the contract. Click on the Klaytn logo in the icon panel. Import an account by clicking the plus button next to `Account`. Make sure that the account has sufficient KLAY to deploy and transact the contracts.

Reviewer: ```suggestion
- Now we can deploy the contract. Click on the Klaytn logo in the icon panel. Import an account by clicking the plus button next to `Account`. Make sure that the account has sufficient KLAY to pay for the transaction of deploying the smart contracts required.
```

![](img/deploy-with-ide/05_deployment_account.png)

- Set Gas limit and Value to send. 
  - You may need to set higher Gas limit if you are deploying a more complicated contract. In this example, you can leave it as it is.
  - Set `Value` to 0 unless you want to send `KLAY` to the contract at the time of deployment.
- Enter "Hello World!" as an argument for constructor function and click on `Deploy` button.

![](img/deploy-with-ide/03_deployment_hello.png)

> - If the contract is successfully deployed, you will see the result in the terminal. 

Reviewer: ```suggestion
- If the contract is successfully deployed, you will see the corresponding transaction receipt and detailed result in the terminal. 
```

> - You can interact with the contract by clicking on the function buttons. The functions are represented in different colors. `constant` or `pure` functions in Solidity have blue bottons (`greet` in the example) and do not create a new transaction, so they don't cost any gas. Red buttons (`kill` in the example) represent `payable` functions that change the state on the blockchain, consumes gas and can accept value. Orange buttons are for `non-payable` functions that change the contract state but do NOT accept a value.

Reviewer: ```suggestion
- You can interact with the contract by clicking on the function buttons. The functions are represented in different colors. `constant` or `pure` functions in Solidity have blue bottons (`greet` in the example) and do not create a new transaction, so they don't cost any gas. Red buttons (`kill` in the example) represent `payable` functions that change the state on the blockchain, consume gas and can accept value. Orange buttons are for `non-payable` functions that change the contract state but do NOT accept a value.
```
